src/Event.cpp

```diff
@@ -866,8 +919,18 @@ double Event::HT(const JetCollection jets) {
 
 double Event::ST(const JetCollection jets, const ParticlePointer lepton, const METPointer met) {
 	// ST = HT + MET + lepton pt
-	double ht = Event::HT(jets);
-	return ht + met->et() + lepton->pt();
+	double ht = 0;
+	if ( jets.size() > 0 ) ht = Event::HT(jets);
```

Comment:
If you are really concerned about having 0 jets, this should got into ```Event::HT``` and not here!